storage/innobase/xtrabackup/test/t/bug1192834-purge.sh

```diff
@@ -0,0 +1,33 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+start_server --innodb_file_per_table
+load_dbase_schema sakila
+load_dbase_data sakila
+
+run_cmd $MYSQL $MYSQL_ARGS -e "DELETE FROM payment LIMIT 100" sakila
+
+sleep 2
+
+backup_dir="$topdir/backup"
+
+innobackupex --no-timestamp --compact $backup_dir
```

Comment:
Do you use `innnobackupex` here intentionally? If yes, please put a comment why, IIRC it is going to be deprecated.